.github/workflows/testing.yml

```diff
@@ -30,39 +30,39 @@ jobs:
     name: Test run action (valid with custom Flake8)
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - run: pip install flake8==3.8.0
+    - uses: actions/checkout@v3
+    - run: pip install flake8==4.0.1
```

Comment:
The version we install isn't important for this test. It is about showing that you can, and how to do it.

I do not feel bumping this every time flake8 makes a new release, so I am happy with any "non-latest" version :)